src/middlewares/patient.js

```diff
@@ -3,18 +3,26 @@
 
     var keyFor = {
         "Observation": "subject",
-        "MedicationPrescription": "patient"
+        "MedicationPrescription": "patient",
+        "Encounter": "patient"
```

Comment:
We can extract from FHIR metadata all refs to Patient

On 3 September 2015 at 16:02, nschwertner <notifications@github.com> wrote:

> In src/middlewares/patient.js
> <https://github.com/FHIR/fhir.js/pull/50#discussion_r38642475>:
>
> > @@ -3,18 +3,26 @@
> >
> >      var keyFor = {
> >          "Observation": "subject",
> > -        "MedicationPrescription": "patient"
> > +        "MedicationPrescription": "patient",
> > +        "Encounter": "patient"
>
> Is there a definitive list of the resources "with a patient" in DSTU2?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/FHIR/fhir.js/pull/50/files#r38642475>.
>